sdk/src/Services/DynamoDBv2/Custom/DocumentModel/DocumentTransactWrite.cs

```diff
@@ -158,6 +161,40 @@ public void AddDocumentToUpdate(Document document, TransactWriteItemOperationCon
             AddDocumentToUpdateHelper(document, TargetTable.MakeKey(document), operationConfig);
         }
 
+        /// <summary>
+        /// Add a single item to update, identified by its hash primary key, using the specified update expression and config.
+        /// </summary>
+        /// <param name="hashKey">Hash key of the document.</param>
+        /// <param name="updateExpression">Update expression to use.</param>
+        /// <param name="operationConfig">Configuration to use.</param>
+        public void AddKeyToUpdate(Primitive hashKey, Expression updateExpression, TransactWriteItemOperationConfig operationConfig = null)
```

Comment:
Similar to https://github.com/aws/aws-sdk-net/pull/3294/files#r1648167618 above, could we keep calling these three `AddDocumentToUpdate`?

Reply:
Done